Sources/StreamVideoSwiftUI/CallView/CallView.swift

```diff
@@ -16,10 +16,12 @@ public struct CallView<Factory: ViewFactory>: View {
     
     var viewFactory: Factory
     @ObservedObject var viewModel: CallViewModel
+    @ObservedObject private var microphoneChecker: MicrophoneChecker
```

Comment:
Why do we need this here? We can just use it directly in the modifier as a StateObject.